python/metatensor-torch/tests/learn/module_map.py

```diff
@@ -111,3 +189,49 @@ def test_torchscript_module_map(single_block_tensor):
     buffer.seek(0)
     torch.jit.load(buffer)
     buffer.close()
+
+
+@pytest.mark.parametrize(
```

Comment:
I'm not sure it is worth parametrizing all these tests on `out_properties`. Just one version with `out_properties` defined to something should be plenty to check this part of the code.